src/environments/environment.prod.ts

```diff
@@ -50,6 +51,7 @@ export const environment: Env = {
   forceSSRServerMode:
     base.forceSSRServerMode === "1" || base.forceSSRServerMode === "true",
   spinnerType: base.spinnerType,
+  multiLang: base.multiLang === "1" || base.multiLang === "true",
```

Comment:
Better name for this option could be `enableMultiLang` or `multiLangEnabled`
(there is not a great consistency over these options, having both `authEnabled` and `enableFooter`... but in any case `enable` somewhere would be useful

Reply:
the choice is yours! "enable" in the name makes sense

Reply:
Yep, let's add an enable somewhere please 🙏